Sorting Algorithms/Quick Sort Algorithm/Java/QuickSort.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Scanner;
+
+/* ===== ===== =====
+
+Theory of Programming
+
+Quick Sort Algorithm
+http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/
+GitHub - https://github.com/VamsiSangam/theoryofprogramming
+Code Contributor - Aditya Aggarwal (github.com/the-cybersapien, cybersapien.xyz)
+
+===== ===== ===== */
+
+public class QuickSort{
+
+	public static void main(String[] args) {
+        //Initialize the Scanner Object to take input
+        Scanner in = new Scanner(System.in);
+        //Get the Size of the Array
+        System.out.println("Enter the size of the array:");
+        int size = in.nextInt();
+        //Initialize the Array for sorting the elements
+        int[] array = new int[size];
+        //Get the data from user
+        System.out.println("Enter the elements of the array: ");
+
+        for (int i = 0; i < size; i++) {
+            array[i] = in.nextInt();
+        }
+
+        //Call the quickSort function to sort the Array
+        quickSort(array, 0, size-1);
+
+        //Print the Sorted Array
+        System.out.println("Sorted Array: ");
+
+        for (int a : array) {
+            System.out.print(a + " ");
+        }
+        System.out.println();
+    }
+
+    private static void quickSort(int[] arr, int low, int high) {
```

Comment:
Since we're only using this function inside this class, I'd say it's better to keep this private.
If you want, I'll fix it.